spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/LimsBindingInitializer.java

```diff
@@ -559,7 +561,7 @@ private Group resolveGroup(Object element) throws IllegalArgumentException {
    */
   private User resolveUser(Object element) throws IllegalArgumentException {
     Long id = null;
-    if (element instanceof String && !"".equals(element)) id = NumberUtils.parseNumber((String) element, Long.class).longValue();
+    if (element instanceof String && !isStringEmptyOrNull((String) element)) id = NumberUtils.parseNumber((String) element, Long.class).longValue();
```

Comment:
Again, out of scope but since you touched the code :smile_cat: 
Does our style guide allow for single line if statements?  :neckbeard: :neckbeard: